templates/nginx/galaxy-main.j2

```diff
@@ -82,6 +74,14 @@ server {
 		alias /;
 		add_header X-Frame-Options SAMEORIGIN;
 		add_header X-Content-Type-Options nosniff;
+
+		# CORS
+		if ($http_origin ~ "^https?://(covid19map\.elixir-luxembourg\.org|www\.bx\.psu\.edu)$") {
```

Comment:
so not websockets to something like that needed?

Reply:
no, websockets aren't needed here, they're not relevant in the context of users browsers making requests, which should always have an origin with http/s